user.css

```diff
@@ -589,6 +597,11 @@ li.GlueDropTarget {
     transition: opacity,left 0.2s ease;
 }
 
+.playback-bar:active .prog-tooltip {
+	opacity: 1;

```

Comment:
would you mind indenting everything with 4 spaces, thanks

Reply:
my bad, was editing this part with npp instead of vs code so it inserted a tab instead of spaces

Reply:
there's another tab above